tests/memoryview/memoryview.pyx

```diff
@@ -1314,5 +1330,5 @@ def test_untyped_index(i):
 
 _PERFORMANCE_HINTS = """
 243:9: Use boundscheck(False) for faster access
-1313:21: Index should be typed for more efficient access
+1329:21: Index should be typed for more efficient access
```

Comment:
This performance hint was different from your example (1329 vs. 1325), perhaps due to code staleness?

Reply:
It's just a line number

Reply:
Yeah, I'm not clear on why @scoder's line number differs from mine. I think L1329 is the correct line based off this current revision

Reply:
Might have been an incomplete manual change on my part. If CI is happy, I'll be happy.